Source/Eto.WinForms/Forms/Printing/PrintDocumentHandler.cs

```diff
@@ -49,12 +49,13 @@ public override void AttachEvent(string id)
 
 		protected virtual void HandlePrintPage(object sender, sdp.PrintPageEventArgs e)
 		{
-			var graphics = new Graphics(new GraphicsHandler(e.Graphics));
-
-			var args = new PrintPageEventArgs(graphics, e.PageBounds.Size.ToEto(), currentPage);
-			Callback.OnPrintPage(Widget, args);
-			currentPage++;
-			e.HasMorePages = currentPage < PageCount;
+			using (var graphics = e.Graphics.ToEto(false))
```

Comment:
this is how it should be done - dispose the disposable if you create it (but don't hurt native graphics by passing `false` to `ToEto`)